.vscodeignore

```diff
@@ -25,3 +25,6 @@ tsconfig.json
 tslint.json
 codebuild-image-buildspec.yml
 node_modules
+docs/**
+designs/**
```

Comment:
@JadenSimon Thanks, awesome improvement! Wonder if we should explicitly allow, instead of hunting down these unwanted things. https://github.com/Microsoft/vscode-vsce/issues/12#issuecomment-285964473

```
# Ignore everything
*
*/**

# Whitelist what you need
!.vscodeignore
!src/*
```